packages/legend-query/src/components/QueryBuilder.tsx

```diff
@@ -208,107 +218,149 @@ export const QueryBuilder = observer(
         <GlobalHotKeys keyMap={keyMap} handlers={handlers}>
           <Backdrop className="backdrop" open={queryBuilderState.backdrop} />
           <div className="query-builder__content">
-            <ResizablePanelGroup orientation="horizontal">
-              <ResizablePanel minSize={120}>
-                {isQuerySupported ? (
-                  <ResizablePanelGroup orientation="vertical">
-                    <ResizablePanel size={450} minSize={300}>
-                      <ResizablePanelGroup orientation="horizontal">
-                        {queryBuilderState.querySetupState.showSetupPanel && (
-                          <ResizablePanel minSize={40} direction={1}>
-                            <QueryBuilderSetupPanel
-                              queryBuilderState={queryBuilderState}
-                            />
-                          </ResizablePanel>
-                        )}
-                        {!queryBuilderState.querySetupState.showSetupPanel && (
-                          <ResizablePanel minSize={40} size={40} direction={1}>
-                            <QueryBuilderSetupPanel
-                              queryBuilderState={queryBuilderState}
-                            />
-                          </ResizablePanel>
-                        )}
-                        <ResizablePanelSplitter />
-                        <ResizablePanel minSize={40} direction={[1, -1]}>
-                          <QueryBuilderExplorerPanel
-                            queryBuilderState={queryBuilderState}
-                          />
-                        </ResizablePanel>
-                        <ResizablePanelSplitter />
-                        {queryBuilderState.showFunctionPanel && (
-                          <ResizablePanel
-                            minSize={40}
-                            direction={
-                              queryBuilderState.showParameterPanel
-                                ? [1, -1]
-                                : -1
-                            }
-                          >
-                            <QueryBuilderFunctionsExplorerPanel
-                              queryBuilderState={queryBuilderState}
-                            />
-                          </ResizablePanel>
-                        )}
-                        {queryBuilderState.showFunctionPanel &&
-                        queryBuilderState.showParameterPanel ? (
-                          <ResizablePanelSplitter />
+            <div className="query-builder__activity-bar">
+              <div className="query-builder__activity-bar__items"></div>
```

Comment:
Can we add a changelog message for this. Important feature for debugging :)

Reply:
of course!